app/presenters/check_answers/other_info_card.rb

```diff
@@ -13,33 +13,44 @@ def initialize(claim)
     def row_data
       [
         {
-          head_key: 'other_info',
-          text: check_missing(claim.other_info.present?) do
-            ApplicationController.helpers.multiline_text(claim.other_info)
+          head_key: 'is_other_info',
```

Comment:
this could potentially use the `process_boolean_value` method to simplify this code. :)